src/api/parser/test/e2e/parser-flow.test.js

```diff
@@ -85,7 +73,17 @@ describe("Testing parser service's flow", () => {
   });
 
   test('Parser should process 2 invalid feeds', async () => {
-    getWikiFeeds.mockReturnValue(invalid);
+    const invalid = [
+      {
+        author: 'John Doe',
+        url: 'https://johnhasinvalidfeed.com/feed',
+      },
+      {
+        author: 'Jane Doe',
+        url: 'https://janehasinvalidfeed.com/feed',
```

Comment:
these urls don't exist/are invalid => response is fast shouldn't affect test's time